### PR TITLE
Routing Added to quiz app

### PR DESCRIPTION
**Pull request by Mahi24-prog**
### Routing Functionality add to Quiz App

https://user-images.githubusercontent.com/59019169/161366617-ddfe1007-ca13-4fd1-abbe-c11af4ca750e.mp4

### CODE REVIEW

File: src/Components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
+        <div className='navbar-left d-flex'>
+          <img src="/images/brand-icon.png" className='brang-icon' />
+          <Link to="/">
             <h1 className='brand-name lg-h'>Jackpot Quiz</h1>
+          </Link>
+        </div>
+        <div className='navbar-right flex-wrap'>
+          <div className='navbar-icon'>
+            <Link to="/rule">
+              <EventNoteIcon style={{ fontSize: "3rem" }} />
```
Comment (prathmeshjagtap) [inline src/Components/Navbar/Navbar.jsx:19]: avoid giving  inline styles

File: src/Routing/Router.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import RulePage from "../Pages/Rule/RulePage"
+
+const Router = () => {
+  return (
+    <>
+        <Routes>
+            <Route exact path='/' element={<Home/>}></Route>
+            <Route exact path='/questions' element={<QuestionsPage/>}></Route>
+            <Route exact path='/result' element={<ResultPage/>}></Route>
+            <Route exact path='/rule' element={<RulePage/>}></Route>
```
Comment (prathmeshjagtap) [inline src/Routing/Router.jsx:15]: you can add one more route to 404 error page not found

**Review by prathmeshjagtap — APPROVED** [review]
Rest everything is good , Just avoid giving  inline styles

**Review by sakleprajwal — APPROVED** [review]
Everything looks good. Pease go ahead

File: src/Routing/Router.jsx (hunk 2 above)
Comment (Mahi24-prog) [reply src/Routing/Router.jsx:15]: ok

**Pull request merged**